sandbox/cri/Dockerfile

```diff
@@ -0,0 +1,105 @@
+    && apt-get -y install --no-install-recommends \
+        binutils-dev \
+        cmake \
+        curl \
+        file \
+        g++ \
+        gcc \
+        git \
+        iperf3 \
+        iproute2 \
+        jq \
+        libcurl4-openssl-dev \
+        libdw-dev \
+        libiberty-dev \
+        libssl-dev \
+        lsof \
+        make \
+        musl-tools \
+        net-tools \
+        openssh-client \
+        pkgconf \
+        zlib1g-dev \
+        libseccomp-dev \
+        btrfs-tools \
+        socat \
+	sudo \
+	iptables \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install the Rust toolchain
+#
+# Install 1.32.0 due to Firecracker crashing with 148 code
+RUN mkdir "$TMP_BUILD_DIR" \
+    && curl https://sh.rustup.rs -sSf | sh -s -- -y \
+	&& rustup install 1.32.0 \
+	&& rustup override set 1.32.0 \
+        && rustup target add x86_64-unknown-linux-musl \
+    && cd /
+
+# Install Firecracker and set binaries to TMP_BUILD_DIR
+RUN git clone https://github.com/firecracker-microvm/firecracker.git
+WORKDIR firecracker
+RUN git checkout tags/v"$FIRECRACKER_VERSION"
+RUN cargo build --release --features vsock --target-dir "$TMP_BUILD_DIR"
+RUN cp "$TMP_BUILD_DIR"/x86_64-unknown-linux-musl/release/firecracker /usr/local/bin
+RUN cp "$TMP_BUILD_DIR"/x86_64-unknown-linux-musl/release/jailer /usr/local/bin
+RUN mkdir /etc/containerd && touch /etc/containerd/config.toml
+
+# Install containerd
+RUN mkdir -p /go/src/github.com/containerd/containerd
+RUN git clone https://github.com/containerd/containerd.git /go/src/github.com/containerd/containerd
+WORKDIR /go/src/github.com/containerd/containerd
+RUN git checkout v1.2.1
+RUN make
+RUN cp bin/* /usr/local/bin
+
+# Install runc
+RUN mkdir -p /go/src/github.com/opencontainers/runc
+RUN git clone https://github.com/opencontainers/runc /go/src/github.com/opencontainers/runc
+WORKDIR /go/src/github.com/opencontainers/runc
+RUN git checkout 96ec2177ae841256168fcf76954f7177af9446eb
+RUN make static BUILDTAGS='seccomp'
+RUN make BINDIR='/usr/local/bin' install
+
+# Build and install firecracker-containerd
+WORKDIR /firecracker-containerd/
+COPY sandbox/cri/config.toml /etc/containerd/config.toml
+COPY sandbox/cri/cri.sh /usr/local/bin
+COPY sandbox/cri/firecracker-runtime.json /etc/containerd/
+RUN mkdir -p /var/lib/firecracker-containerd/runtime
+# Retrieve kernel image and root drive
+RUN curl -fsSL -o /var/lib/firecracker-containerd/runtime/hello-vmlinux.bin https://s3.amazonaws.com/spec.ccfc.min/img/hello/kernel/hello-vmlinux.bin
+RUN curl -fsSL -o /var/lib/firecracker-containerd/runtime/hello-rootfs.ext4 https://s3.amazonaws.com/spec.ccfc.min/img/hello/fsfiles/hello-rootfs.ext4
+RUN make all && make install
+RUN mkdir /var/run/firecracker-containerd
+
+# Install CRI
+WORKDIR /
+# CRI does not have support for Go modules yet.
+ENV GOPATH="/go"
+RUN go get github.com/containerd/cri
+RUN go get -u github.com/onsi/ginkgo/ginkgo
+WORKDIR /go/src/github.com/containerd/cri
+RUN make install.deps
```

Comment:
`install.deps` [installs](https://github.com/containerd/cri/blob/master/hack/install/install-deps.sh) containerd, runc, cni and other dependencies needed for cri. Do we really need to keep both `install.deps` and our own logic for installing deps ?